packit_service/worker/whitelist.py

```diff
@@ -190,8 +191,17 @@ def check_and_report(
         :param project: GitProject
         :return:
         """
+
+        # whitelist checks dont apply to CentOS (Pagure)
+        if isinstance(
+            event,
+            (PushPagureEvent, PullRequestPagureEvent, PullRequestCommentPagureEvent),
+        ):
+            logger.info("Centos (Pagure) events dont require whitelist checks")
```

Comment:
We need to think about this more when working on the Pagure support as upstream (#556).

Reply:
yes, i am aware of that, and in future we can achieve CentOS stream events exclusion using some Event attribute 

e.g. our centos listener is adding topic field to message dictionary which is sent for processing. it consists of sourceurl+gitevent -> sourceurl can be used for this purpose 

i think that at this point, there is no reason to implement it.

Reply:
should i implement it in this PR?

Reply:
Not need to solve that here.